features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,16 @@
-# put step definitions in this file
\ No newline at end of file
+# put step definitions in this file
+Given(/^I visit the site\.$/) do
+  visit '/'
+end
+
+Then(/^I want to see that Fabian is a student at Craft Academy$/) do
+  expect(page).to have_content
+end
```

Comment:
This step doesn't test anything. If you are following the cycle correctly, you should see a test *fail* before you write the code to make it pass.

Write the step `Then I want to see "Fabian" is a student at Craft Academy`. It will give you a step definition and you'll store `"Fabian"` in a variable and expect the page to `have_content <variable>`